runners/flink/job-server/test_pipeline_jar.sh

```diff
@@ -43,6 +43,16 @@ case $key in
         shift # past argument
         shift # past value
         ;;
+    --python_container_image)
```

Comment:
Lets just take the image name in standard form `imageName:tag`

Reply:
Done